util.py

```diff
@@ -164,6 +169,26 @@ def is_valid_hostname(hostname):
     return True
 
 
+def check_yaml(conf_path):
+    with open(conf_path) as f:
+        check_config = yaml.load(f.read(), Loader=yLoader)
+        assert 'init_config' in check_config, "No 'init_config' section found"
```

Comment:
You should make sure that those AssertErrors would be caught when calling check_yaml, you don't want the agent to crash on that